dist/assembly.xml

```diff
@@ -161,6 +161,30 @@
       <useTransitiveFiltering>false</useTransitiveFiltering>
     </dependencySet>
 
+    <!-- Cassandra module -->
+    <dependencySet>
+      <outputDirectory>${hawkular.dist.zip.root.dir}/modules/org/apache/cassandra/main</outputDirectory>
+      <includes>
+        <include>org.apache.cassandra:cassandra-all:jar</include>
```

Comment:
Indeed, for some reason, I thought I needed access to a class or two inside this jar, but I was wrong. I'm going to remove this and update the PR.